### DISCUSSION TITLE
Not finding file of an include

### DISCUSSION THREAD

**Discussion by javihernant** [neovim/neovim#27565] [Q&A]
I believe my problem is: I am at `src/index.ts` and, inside that file there is a `include ./file.ts`. I think neovim thinks it has to look for it in `src` (where nvim was opened), instead of inside `src/` (`index.ts` is opened there). I'm using lsp with deno-ls.
![Screenshot](https://github.com/neovim/neovim/assets/73640929/0a5137c0-5ad9-4efb-a8c3-54e2f144dcc6)

**Comment by clason** [comment]
That's not Neovim; that's whatever language server you are using. Please check its documentation.
